### DISCUSSION TITLE
Kudos is needed!

### DISCUSSION THREAD

**Discussion by MarkAC007** [blakeblackshear/frigate#13300] [General]
I just want to say thank you very much for the latest release of frigate. The changes in this release are huge and I'm already enjoying exploring the new features and functionality.   Great effort indeed.

**Comment by simondsmason** [comment]
I second this sentiment.  The release is amazing.  I went from windowing my camera feeds on my touch screens to exposing the new UI in it's entirety.  Congratulations to the whole team for an amazing delivery.  

@blakeblackshear if appropriate please tag the people who are responsible for this.

**Reply by ElectronicBattle** [reply]
ditto: it's astonishing work and it's become an addiction to play with!

**Comment by dsokoloski** [comment]
Agreed!  Wow, just amazing work!  What a vast improvement!

**Reply by Lulugaot** [reply]
Thank you ! The latest version has worked flawlessly since I updated it when I came back from vacation on August 14. I am using a Pi 5 , 8 GB, official Pi NVME hat with a Corsair 2230 nvme M.2 with 1 TB of storage, a Google Coral USB all mounted to an Amazon purchased OONO enclosure. I struggled with Home Assistant, YAML, and Frigate since starting this journey in the last few days of May on my old, original Pi 3. Once setup completed early June, it stuttered and I bought the Google Coral and put it aside, over the next few days / weeks I purchased the Pi 5 , official active cooler, official case, HAT , and NVME SSD. I watched YouTube videos and reviewed online documentation for setup and configuration due to issues here and there. It worked reasonably well with occasional hiccups. Finally after updating to latest version after holding off because I thought it would break my setup I pulled the plug and updated. WOW, it works beautifully, is apparently stable and is feature rich. It has been running every since. I use Reolink cameras, one is H264 and the other is H265. The H265 does occasionally give me a little random green pixelation issues in places on the video images but I can live with that since it runs fine otherwise. Back in June with my original setup both H264 and H265 worked well then one day the H265 started having issues and did not work. Many changes later it worked some of the time. As said above, now with this update everything works (green random pixelation once in a blue moon on the image of that camera) but it is kind of rare but I can live with that. Sorry for the long post. Appreciate all the hard work from everyone involved.

**Comment by Lulugaot** [comment]
![PXL_20240830_213947917](https://github.com/user-attachments/assets/4b94e257-8dba-4557-9d8f-64f7823d00bb)
![PXL_20240830_214120030](https://github.com/user-attachments/assets/76a2a3cb-f519-46be-9cae-b8864faa75c7)
![PXL_20240830_214038813](https://github.com/user-attachments/assets/86700834-c4ef-48a9-8a90-2b19887d8c44)

**Comment by Lulugaot** [comment]
Thank you ! The latest version has worked flawlessly since I updated it when I came back from vacation on August 14. I am using a Pi 5 , 8 GB, official Pi NVME hat with a Corsair 2230 nvme M.2 with 1 TB of storage, a Google Coral USB all mounted to an Amazon purchased OONO enclosure. I struggled with Home Assistant, YAML, and Frigate since starting this journey in the last few days of May on my old, original Pi 3. Once setup completed early June, it stuttered and I bought the Google Coral and put it aside, over the next few days / weeks I purchased the Pi 5 , official active cooler, official case, HAT , and NVME SSD. I watched YouTube videos and reviewed online documentation for setup and configuration due to issues here and there. It worked reasonably well with occasional hiccups. Finally after updating to latest version after holding off because I thought it would break my setup I pulled the plug and updated. WOW, it works beautifully, is apparently stable and is feature rich. It has been running every since. I use Reolink cameras, one is H264 and the other is H265. The H265 does occasionally give me a little random green pixelation issues in places on the video images but I can live with that since it runs fine otherwise. Back in June with my original setup both H264 and H265 worked well then one day the H265 started having issues and did not work. Many changes later it worked some of the time. As said above, now with this update everything works (green random pixelation once in a blue moon on the image of that camera) but it is kind of rare but I can live with that. Sorry for the long post. Appreciate all the hard work from everyone involved.


Lulugaot
4 minutes ago
PXL_20240830_213947917
PXL_20240830_214120030
PXL_20240830_214038813

**Reply by ElectronicBattle** [reply]
That's a neat solution: when I eventually get round to building my own dedicated NVR for 8 wireless IP cameras, this might be one of the hardware platforms I explore.

**Reply by Lulugaot** [reply]
Thank you ! Yes, it works just fine. I use it all the time now to view my Frigate setup with my Reolink cameras while occasionally using my Reolink 8 channel NVR system which I bought and installed back in March of this year. Plus, it is so small (backup system) just in case, but I love using it. My Pi 3 is now setup as my NAS with two 2.5 inch SSD's running only Open Media Vault as network storage and backup to home assistant which runs on the Pi 5 above.

**Comment by Lulugaot** [comment]
Here is a picture
![PXL_20240830_231731926](https://github.com/user-attachments/assets/0e57fe5b-a815-4df8-8219-ac7525b21bfd)
![PXL_20240830_231703084](https://github.com/user-attachments/assets/8fba5295-0568-479a-a498-60968a90413e)
